Sources/Datadog/DatadogCore/Context/DatadogContextProvider.swift

```diff
@@ -88,7 +85,7 @@ internal final class DatadogContextProvider {
     ///
     /// - Parameter receiver: The receiver closure.
     func publish(to receiver: @escaping ContextValueReceiver<DatadogContext>) {
-        queue.async(flags: .barrier) { self.receivers.append(receiver) }
+        queue.async { self.receivers.append(receiver) }
```

Comment:
I made sure this was a leftover - this queue is not `.concurrent`, so using `.barrier` makes no effect.